src/librbd/operation/RenameRequest.cc

```diff
@@ -66,7 +66,10 @@ bool RenameRequest<I>::should_complete(int r) {
                 << "r=" << r << dendl;
   r = filter_state_return_code(r);
   if (r < 0) {
-    lderr(cct) << "encountered error: " << cpp_strerror(r) << dendl;
+    if (r == -EEXIST)
+      ldout(cct, 1) << "RenameRequest: image already exists" << dendl;
```

Comment:
Minor: you don't need to put the class name in the log message -- it's inserted automatically.